resources/assets/js/plugins/i18n.js

```diff
@@ -5,16 +5,16 @@ import fr from './fr.json'
 import es from './es.json'
 import ar from './ar.json'
 import de from './de.json'
+import ja from './ja.json'
 import pt_BR from './pt-br.json'
 import it from './it.json'
 import sr from './sr.json'
 import nl from './nl.json'
+import ko from './ko.json'
 import lv from './lv.json'
 import sv from './sv.json'
-import sk from './sk.json'
```

Comment:
Same thing here, I think you might have removed this language while trying to fix the conflict.

Reply:
yes, i have update it. thank you!